symengine/functions.h

```diff
@@ -924,6 +924,30 @@ class Erf : public OneArgFunction
 //! Canonicalize Erf:
 RCP<const Basic> erf(const RCP<const Basic> &arg);
 
+class Erfc : public OneArgFunction
+{
+    /*   The Gauss error function. This function is defined as:
+     *
+     *   .. math::
+     *      \mathrm{erfc}(x) =1 neg( \frac{2}{\sqrt{\pi}} \int_0^x e^{-t^2})
```

Comment:
Should `neg` be repleaced with `-`?

Also, the parentheses should probably be removed. Like this:

```
\mathrm{erfc}(x) = 1 - \frac{2}{\sqrt{\pi}} \int_0^x e^{-t^2}
```